config/routes.rb

```diff
@@ -56,10 +56,17 @@
   #     resources :products
   #   end
 
+  resources :friendships
+
+  get 'friendship/my_friends' => 'friendships#my_friends', as: :my_friends
+
+  get 'friendship/my_friends/:id' => 'friendships#grant_request', as: :grant_request
```

Comment:
Line is too long. [84/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.